scripts/data/package.json

```diff
@@ -1,20 +1,36 @@
 {
   "name": "data",
   "version": "1.0.0",
-  "description": "",
-  "main": "updateContentOverview.ts",
-  "directories": {
-    "lib": "lib"
-  },
-  "scripts": {},
+  "private": true,
   "author": "",
   "license": "ISC",
```

Comment:
Thats new.. we usually do MIT or Apache2

Reply:
nice, well spotted! I think MIT should do here as well

Reply:
Changed to Apache, in line with the repo itself